addons/docs/README.md

```diff
@@ -1,29 +1,92 @@
+<center>
+  <img src="docs/media/hero.png" width="100%" />
+</center>
+
 # Storybook Docs
 
-Living documentation for your components.
+Storybook Docs transforms your Storybook stories into world-class component documentation.
 
-- [Sneak peak article](https://medium.com/storybookjs/storybook-docs-sneak-peak-5be78445094a)
-- [Technical preview guide](https://docs.google.com/document/d/1un6YX7xDKEKl5-MVb-egnOYN8dynb5Hf7mq0hipk8JE/edit?usp=sharing)
+**DocsPage.** Out of the box, all your stories get a `DocsPage`. `DocsPage` is a zero-config aggregation of your component stories, text descriptions, docgen comments, props tables, and code examples into simple, easy-to-read pages.
 
-## View layer support
+**MDX.** If you want more control, `MDX` allows you to write long-form markdown documentation and stories in one file. You can also use it to write pure documentation pages and embed them inside your Storybook alongside your stories.
 
-Docs supports all view layers that Storybook supports except for React Native (currently). There are some view-layer specific
-features as well. This chart captures the current state of support
+Just like Storybook, Docs supports every major view layer including React, Vue, Angular, HTML, Web components, Svelte, and many more.
+
+Read on to learn more:
+
+- [DocsPage](#docspage)
+- [MDX](#mdx)
+- [Framework support](#framework-support)
+- [Installation](#installation)
+- [Preset options](#preset-options)
+- [Manual configuration](#manual-configuration)
+- [More resources](#more-resources)
+
+## DocsPage
+
+When you [install docs](#installation), every story gets a `DocsPage`. `DocsPage` pulls information from your stories, components, source code, and story metadata to construct a sensible, zero-config default.
```

Comment:
Capitalize Docs because it's a proper noun in this usage. "Storybook Docs" is the feature name.